pkg/virt-handler/virtwrap/api/schema.go

```diff
@@ -215,6 +223,16 @@ type DomainSpec struct {
 	Clock    *Clock       `xml:"clock,omitempty"`
 	Resource *Resource    `xml:"resource,omitempty"`
 	QEMUCmd  *Commandline `xml:"qemu:commandline,omitempty"`
+	Metadata Metadata     `xml:"metadata,omitempty"`
+}
+
+type Metadata struct {
+	GracePeriod GracePeriodMetadata `xml:"http://kubevirt.io graceperiod,omitempty"`
```

Comment:
Can we model this the same way, it is done on the cluster metadata?

```golang
type Metadata struct {
	DeletionTimestamp *Time
	DeletionGracePeriodSeconds *int64
}
```

Reply:
Yeah, I'll change this to make it consistent with whatever those values are named.

To be consistent with the pod's, it's actually terminationGracePeriodSeconds and not deletionGracePeriodSeconds

Reply:
Hm, in the metadata section it is called `DeletionGracePeriodSeconds`. That was the one I was referring to.

Reply:
Ah, i see